core/src/test/java/org/testcontainers/dockerclient/ImagePullTest.java

```diff
@@ -0,0 +1,58 @@
+package org.testcontainers.dockerclient;
+
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+
+public class ImagePullTest {
+
+    @Test
+    public void pullOfficialLatestImageTest() {
+        doPullStartAndStop("alpine:latest");
+    }
+
+    @Test
+    public void pullOfficialImageByTagTest() {
+        doPullStartAndStop("alpine:3.6");
+    }
+
+    @Test
+    public void pullOfficialImageByShaTest() {
+        doPullStartAndStop("alpine@sha256:8fd4b76819e1e5baac82bd0a3d03abfe3906e034cc5ee32100d12aaaf3956dc7");
+    }
+
+    @Test
+    public void pullLatestImageTest() {
+        doPullStartAndStop("gliderlabs/alpine:latest");
+    }
+
+    @Test
+    public void pullImageByTagTest() {
+        doPullStartAndStop("gliderlabs/alpine:3.5");
+    }
+
+    @Test
+    public void pullImageByShaTest() {
+        doPullStartAndStop("gliderlabs/alpine@sha256:a19aa4a17a525c97e5a90a0c53a9f3329d2dc61b0a14df5447757a865671c085");
+    }
+
+    @Test
+    public void pullLatestImageFromPublicRegistryTest() {
+        doPullStartAndStop("quay.io/coreos/etcd:latest");
```

Comment:
do we use this image already? If not, to improve the tests speed, we should use already used images (less downloads)